pages/resilience.js

```diff
@@ -1,24 +1,199 @@
 // @author Jimmy Yang
 import Banner from '../components/shared/Banner'
 import TextAndPic from '../components/shared/TextAndPic'
+import ColourTitle from '../components/ResilienceResource/ColourTitle'
+import ColourBox from '../components/ResilienceResource/ColourBox'
+import ColourBoxContainer from '../components/ResilienceResource/ColourBoxContainer'
 
-function resilience() {
+function resilience({ internalResilience, copingPursuits, externalResilience }) {
```

Comment:
let's keep it consistent and name components with Captial

Reply:
update: realized all the components in `pages/` were lowercase, ignore this comment